src/SupportMessage/SupportMessage.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC } from 'react'
+import styled, { css } from 'styled-components'
+import { lighten } from 'polished'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { theme, Color } from '../theme'
+
+const styles = {
+  info: {
+    iconColor: 'secondary',
+    backgroundColor: theme.colors.background,
+    icon: 'info',
+  },
+  'info-outline': {
+    iconColor: 'secondary',
+    backgroundColor: theme.colors.white,
+    icon: 'info',
+  },
+  alert: {
+    iconColor: 'agentWarning',
+    backgroundColor: theme.colors.bgSecondary,
+    icon: 'alert',
+  },
+  warning: {
+    iconColor: 'error',
+    backgroundColor: lighten(0.45, theme.colors.error),
+    icon: 'warning',
+  },
+}
+
+export type SupportMessageProps = {
+  className?: string
+  description: string
+  type: 'info' | 'info-outline' | 'alert' | 'warning'
+  title?: string
+  borderColor?: string
```

Comment:
`borderColor` shouldn't be customisable I think? It should be mapped with the type?

EDIT: It seems it's not used at all now, you can remove it I think?